Examples/Swift/ViewController.swift

```diff
@@ -316,6 +316,8 @@ class ViewController: UIViewController, MGLMapViewDelegate, CLLocationManagerDel
     }
     
     func mapView(_ mapView: MGLMapView, didFinishLoading style: MGLStyle) {
+        self.mapView?.localizeLabels()
```

Comment:
Is self needed here?

Reply:
Yes, because `localizesLabels()` is defined on NavigationMapView, not MGLMapView. The MGLMapView-typed `mapView` argument takes precedence, so we have to either use the NavigationMapView-typed property or cast the argument to a NavigationMapView.